cdap-docs/included-applications/source/etl/plugins/transforms/validator.rst

```diff
@@ -58,7 +58,7 @@ Example response::
                                 };"
         }
       }
-      
+
```

Comment:
I don't know why we started putting the descriptions after the code samples. Seems to me that should be reversed, no? That's why rst has the "::" syntax, in that so often it was used.